### DISCUSSION TITLE
How to create templated pages to show details for an item?

### DISCUSSION THREAD

**Discussion by lfkpoa** [observablehq/framework#1577] [General]
Hello.

We would like to create (sub)pages to show details for an item or a situation in a way that the page receives the id or name somehow and then filters the data and shows the details for that item.

Usually there is a page that shows all the data and allows the user to have a greater picture and to select an item to analyse the details, which would link to the detail subpage, passing the id.

Evidence framework uses [Templated Pages](https://docs.evidence.dev/core-concepts/templated-pages/).

Is there a way to do something similar with Observable Framework?

Thank you

**Comment by mbostock** [comment]
This is under active development. Please upvote https://github.com/observablehq/framework/issues/245 https://github.com/observablehq/framework/pull/1523! 🙏 

In the meantime, consider using query parameters and URLSearchParams to link to filtered views.
